Command not working propertly

commands not worrking its from your github

thats my code:


    class SampleCommands(PythonCommandExecutor):
        commands = [
            PyCommand('sample', 'sampleCommand', 'sampleOnTabComplete')
        ]
    
        def sampleCommand(self, sender, command, label, args):
            # Print to console
            self.plugin.logger.info('aaaa')
            # Print message to command sender
            sender.sendMessage(command.getName())
    
            # For `/sample test` send "TEST"
            if args and args[0] == "test":
                sender.sendMessage("TEST")
            return True
    
        def sampleOnTabComplete(self, sender, command, alias, args):
            return ['test']
    class SamplePluginMainClass(PythonPlugin):
    
        def onEnable(self):
            # Add commands
            self.apply_command_executor(SampleCommands)
            self.logger.info("plugin enabled!")
    
        def onDisable(self):
            self.logger.info("plugin disabled!")

thats my plugin.yml

    name: SamplePlugin
    main: SamplePluginMainClass
    version: 0.1-dev
    commands:
      sample:
        description: send a sample message
        usage: /<command>
            


thats my error:


    [19:56:05 INFO]: marcel1842 issued server command: /sample
    [19:56:05 ERROR]: null
    org.bukkit.command.CommandException: Unhandled exception executing command 'sample' in plugin SamplePlugin v0.1-dev
	    at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at org.bukkit.craftbukkit.v1_17_R1.CraftServer.dispatchCommand(CraftServer.java:790) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at net.minecraft.server.network.PlayerConnection.handleCommand(PlayerConnection.java:1931) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1770) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1751) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:46) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:1) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at net.minecraft.network.protocol.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:30) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at net.minecraft.server.TickTask.run(SourceFile:18) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at net.minecraft.util.thread.IAsyncTaskHandler.executeTask(SourceFile:151) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at net.minecraft.util.thread.IAsyncTaskHandler.executeNext(SourceFile:125) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at net.minecraft.server.MinecraftServer.bf(MinecraftServer.java:1148) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at net.minecraft.server.MinecraftServer.executeNext(MinecraftServer.java:1141) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at net.minecraft.util.thread.IAsyncTaskHandler.executeAll(SourceFile:110) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at net.minecraft.server.MinecraftServer.sleepForTick(MinecraftServer.java:1124) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1054) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:305) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    at java.lang.Thread.run(Thread.java:833) [?:?]
    Caused by: org.python.core.PyException: AttributeError: 'PyCommand' object has no attribute 'executer'
	    at org.python.core.Py.AttributeError(Py.java:178) ~[?:?]
	    at org.python.core.PyObject.noAttributeError(PyObject.java:965) ~[?:?]
	    at org.python.core.PyObject.__getattr__(PyObject.java:959) ~[?:?]
	    at org.python.pycode._pyx9.onCommand$7(<iostream>:82) ~[?:?]
	    at org.python.pycode._pyx9.call_function(<iostream>) ~[?:?]
	    at org.python.core.PyTableCode.call(PyTableCode.java:173) ~[?:?]
	    at org.python.core.PyBaseCode.call(PyBaseCode.java:306) ~[?:?]
	    at org.python.core.PyBaseCode.call(PyBaseCode.java:197) ~[?:?]
	    at org.python.core.PyFunction.__call__(PyFunction.java:485) ~[?:?]
	    at org.python.core.PyMethod.instancemethod___call__(PyMethod.java:237) ~[?:?]
	    at org.python.core.PyMethod.__call__(PyMethod.java:228) ~[?:?]
	    at org.python.core.PyMethod.__call__(PyMethod.java:218) ~[?:?]
	    at org.python.core.PyMethod.__call__(PyMethod.java:213) ~[?:?]
	    at org.python.core.PyObject._jcallexc(PyObject.java:3565) ~[?:?]
	    at org.python.core.PyObject._jcall(PyObject.java:3598) ~[?:?]
	    at org.python.proxies.__main__$PythonCommandExecutor$1.onCommand(Unknown Source) ~[?:?]
	    at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[spigot-1.17.1.jar:3284-Spigot-3892929-0ebef35]
	    ... 19 more

--- a/src/main/resources/framework/command_executor.py
+++ b/src/main/resources/framework/command_executor.py
@@ -78,7 +78,7 @@ class PythonCommandExecutor(CommandExecutor_, TabCompleter_):
         if isfunction(command_.executor):
             return command_.executor(sender, command, label, args)
 
-        executor = getattr(self, command_.executer)
+        executor = getattr(self, command_.executor)
         return executor(sender, command, label, args)
 
     def onTabComplete(self, sender, command, alias, args):

